package.json

```
@@ -34,7 +34,7 @@
     "@types/node": "~14.14.22",
     "@typescript-eslint/eslint-plugin": "~4.14.1",
     "@typescript-eslint/parser": "~4.14.1",
-    "@umb-network/toolbox": "~4.1.1",
+    "@umb-network/toolbox": "5.0.0-alpha.1",
```

```suggestion
    "@umb-network/toolbox": "5.0.0",
```

I released non alfa, so we can update.